src/duckling/splash/splash.component.ts

```diff
@@ -0,0 +1,146 @@
+import {
+    OnInit,
+    Component,
+    Input,
+    Output,
+    EventEmitter,
+    NgZone
+} from 'angular2/core';
+import {MD_LIST_DIRECTIVES} from '@angular2-material/list';
+import {remote} from 'electron';
+
+import {JsonLoaderService} from '../util/json-loader.service';
+import {PathService} from '../util/path.service';
+import {DialogService} from '../util/dialog.service';
+
+interface ProjectModel {
+    title : string,
+    path : string
+}
+
+@Component({
+    selector: 'dk-splash-screen',
+    directives: [
+        MD_LIST_DIRECTIVES
+    ],
+    styleUrls: [ './duckling/splash/splash.component.css' ],
+    template: `
+        <div class="splash-screen">
+            <div class="left-section">
+                <md-nav-list>
+                    <md-list-item
+                    md-list-item
+                    *ngFor="#project of projects"
+                    (click)="openProject({title: project.title, path: project.path})">
+                        <p md-line class="project-title"> {{project.title}} </p>
+                        <p md-line class="project-path"> {{project.path}} </p>
+                    </md-list-item>
+                </md-nav-list>
+
+                <div class="actions">
+                    <a (click)="onNewProjectClick($event)">
+                        <i class="fa fa-file-o" aria-hidden="true"></i>
+                        New
+                    </a>
+                </div>
+            </div>
+
+            <div class="right-section">
+                <div class="duckling-title">
+                    <div class="duckling-name">
+                        Duckling
+                    </div>
+                    <div class="duckling-version">
+                        {{version}}
+                    </div>
+                </div>
+            </div>
+
+            <div class="the-duck"></div>
+        </div>
+    `
+})
+export class SplashComponent implements OnInit {
+    private version : string = "0.0.1";
+    private projects : ProjectModel[] = [];
+    private dialogOptions : {};
+    private MAX_ENTRIES : number = 8;
+
+    @Output()
+    projectOpened : EventEmitter<ProjectModel> = new EventEmitter();
+
+    constructor(private jsonLoader : JsonLoaderService,
+                private path : PathService,
+                private dialog : DialogService,
+                private zone : NgZone) {
```

Comment:
zone is not used anywhere in the splash component.  That code was already extracted into a service.